src/functions/authoriser/application/verifyEmployeeId.ts

```diff
@@ -0,0 +1,37 @@
+import { DynamoDB } from 'aws-sdk';
+import { VerifiedTokenPayload } from '../application/AdJwtVerifier';
+
+export default async function verifyEmployeeId(verifiedToken: VerifiedTokenPayload): Promise<void> {
```

Comment:
We don't seem to be returning anything from this function, the name for me suggests we should expect a boolean?
In the handler are we simply relying on this function throwing an error to determine if the employeeId was verified?

Reply:
We don't return anything because of the `try catch` that we use in the handler will catch the error and act on it. Because checking that is enough to determine the result of the verification.

We can return a boolean which indicates the validity of the id, but in that case we need to check that with an `if else` in the handler.

I'm happy either way, just highlighting the possible outcomes.